constants.xml

```diff
@@ -262,6 +262,7 @@
         <ENABLE_REPOPULATE>true</ENABLE_REPOPULATE>
         <REPOPULATE_STATE_IN_DS>0</REPOPULATE_STATE_IN_DS>
         <REPOPULATE_STATE_PER_N_DS>10</REPOPULATE_STATE_PER_N_DS>
+        <NUM_STORE_TX_BODIES>5</NUM_STORE_TX_BODIES>
```

Comment:
Change NUM_STORE_TX_BODIES to STORE_TX_BODIES_BLK_INTERVAL will be more understandable